vars/neoDeploy.groovy

```diff
@@ -75,10 +71,7 @@ void call(parameters = [:]) {
                     dockerOptions: configuration.dockerOptions
                 ) {
 
-                    neo.verify(this, configuration)
-                    java.verify(this, configuration)
-
-                    String neoExecutable = neo.getToolExecutable(script, configuration)
+                    String neoExecutable = 'neo.sh'
```

Comment:
Does not need to be a parameter of NeoCommandHelper anymore.

Reply:
agreed, I will adapt the code accordingly.

Reply:
done